### DISCUSSION TITLE
ANT media server dashboard is not accessible

### DISCUSSION THREAD

**Discussion by sachingoyal01** [ant-media/Ant-Media-Server#6070] [Q&A]
I installed the ANT media server from AWS marketplace(https://aws.amazon.com/marketplace/pp/prodview-464ritgzkzod6) as per all the instructions given in video https://www.youtube.com/watch?v=EH6v-yUyzjU, but still I am not able to access the ANT media server dashboard with publicIP:5080

**Comment by yashtandon113** [comment]
HI @sachingoyal01 
Could you please SSH into the server, run the below command and share the output?

`sudo service antmedia status`

Also, please make sure that you have subscribed to the AMS Enterprise product on the marketplace.

**Reply by zp181082ksv** [reply]
Hello! I have the same problem!  I am not able to access the ANT media server dashboard with publicIP:5080. I install ANT media on Linode.

**Comment by sachingoyal01** [comment]


**Reply by yashtandon113** [reply]
@sachingoyal01 
Is it possible if you can share the SSH access with your server? You can share it at yash.tandon@antmedia.io.
